platform/linux-generic/odp_event.c

```diff
@@ -59,17 +59,14 @@ int odp_event_type_multi(const odp_event_t event[], int num,
 	return i;
 }
 
-/* For now ODP generic does not support flow awareness,
- * so all flow ids are zero. */
-uint32_t odp_event_flow_id(odp_event_t event ODP_UNUSED)
+uint32_t odp_event_flow_id(odp_event_t event)
```

Comment:
has it be uint64_t everywhere now?

Reply:
The problem was that if ID is 32 bits, max count must be at least 33 bits. I'll change capa from max count to max ID, so that both can be the same size (uint32_t).

Reply:
v3 changes capa/config to max_flow_id. When max_flow_id is 0, flow aware mode is not supported.